docs/layers.md

```diff
@@ -151,8 +149,10 @@ Combination of OpenStreetMap administrative boundaries (zoom >= 8) and Natural E
 
 #### Boundaries properties (common optional):
 
-* `id:left`: For the relation on the left side of the boundary line.
-* `id:right`: For the relation on the right side of the boundary line.
+* `name`: A suggested label, formed from the left and right region names, if available. If the name appears to be too long to be rendered on the geometry at a particular zoom it may be omitted.
```

Comment:
The `name` and `id` are still common properties, we just strip them at low zooms. I like the new description text, but please move them back to the `common` section. (This will better match roads and other layer documentation.)

Reply:
OK, cool. Fixed in 4a01b1d.